src/ray/protobuf/gcs.proto

```diff
@@ -94,6 +94,27 @@ message TaskTableData {
   Task task = 1;
 }
 
+message ActorDeathCause {
+  oneof context {
+    CreationTaskFailureContext creation_task_failure_context = 1;
```

Comment:
Why don't we rename it to ActorInitFailure? Technically runtime env setup failure also is creation task failure.

Reply:
No, creation task is executed after the runtime env is ready. Runtime env setup is in the worker process creation step.